packages/lambda/src/index.js

```diff
@@ -1,11 +1,19 @@
 import fs from 'fs'
+import path from 'path'
 import LambdaChromeLauncher from './launcher'
 import { debug } from './utils'
 import DEFAULT_CHROME_FLAGS from './flags'
 
 const DEVTOOLS_PORT = 9222
 const DEVTOOLS_HOST = 'http://127.0.0.1'
 
+const nssSubPath = fs.readFileSync(path.join(__dirname, 'nss', 'latest'), 'utf8').trim();
```

Comment:
We might need to scope this NSS setup within a `if (process.env.AWS_EXECUTION_ENV)` block, since we probably don't need to mess around with NSS locally, in a non-Lambda context.